charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -104,6 +104,7 @@ spec:
         - --disable-cert-rotation={{ or .Values.audit.disableCertRotation .Values.externalCertInjection.enabled }}
         - --external-data-provider-response-cache-ttl={{ .Values.externaldataProviderResponseCacheTTL }}
         - --enable-k8s-native-validation={{ .Values.enableK8sNativeValidation }}
+        - --enable-constraint-label-metrics={{ .Values.enableConstraintLabelMetrics }}
```

Comment:
This flag should be set conditionally only when values.yaml exclusively include the variable rather than auto setting this.

Reply:
Updated